src/x509.c

```diff
@@ -7565,6 +7565,45 @@ int wolfSSL_X509_REVOKED_get_serial_number(RevokedCert* rev,
     return WOLFSSL_SUCCESS;
 }
 
+/* Retrieve the revocation date from RevokedCert
+ * return WOLFSSL_SUCCESS on success
+ */
+int wolfSSL_X509_REVOKED_get_revocationDate(RevokedCert* rev,
+    byte* in, int* inOutSz)
+{
+    char tmp[MAX_DATE_SIZE];
+
+    WOLFSSL_ENTER("wolfSSL_X509_REVOKED_get_revocationDate");
+
+    if ((rev == NULL) || (in == NULL) || (inOutSz == NULL)) {
+        return (BAD_FUNC_ARG);
```

Comment:
No need for parentheses here and elsewhere when returning an error, NULL, etc. Not a problem, just not our coding style.